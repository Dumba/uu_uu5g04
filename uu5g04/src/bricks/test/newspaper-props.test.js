/**
 * Copyright (C) 2019 Unicorn a.s.
 *
 * This program is free software; you can use it under the terms of the UAF Open License v01 or
 * any later version. The text of the license is available in the file LICENSE or at www.unicorn.com.
 *
 * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
 * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See LICENSE for more details.
 *
 * You may contact Unicorn a.s. at address: V Kapslovne 2767/2, Praha 3, Czech Republic or
 * at the email: info@unicorn.com.
 */

import UU5 from "uu5g04";
import "uu5g04-bricks";

const { mount, shallow, wait } = UU5.Test.Tools;

const CONFIG = {
  mixins: [
    "UU5.Common.BaseMixin",
    "UU5.Common.ElementaryMixin",
    "UU5.Common.SectionMixin",
    "UU5.Common.ColorSchemaMixin",
    "UU5.Common.NestingLevelMixin",
    "UU5.Common.PureRenderMixin",
    "UU5.Common.ContentMixin",
    "UU5.Common.LevelMixin"
  ],
  props: {
    columnsCount: {
      values: [1, 2, 3, 4, 5, 6]
    }
  },
  requiredProps: {},
  opt: {
    shallowOpt: {
      disableLifecycleMethods: false
    }
  }
};

describe(`UU5.Bricks.Newspaper`, () => {
  UU5.Test.Tools.testProperties(UU5.Bricks.Newspaper, CONFIG);
});

describe(`UU5.Bricks.Newspaper docKit examples`, () => {
  it(`UU5.Bricks.Newspaper should render without crash`, () => {
    const wrapper = shallow(
      <UU5.Bricks.Container id={"iiROOT"}>
        <UU5.Bricks.Newspaper id={"uuID"} columnsCount={3}>
          Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas libero. Aliquam erat volutpat. Duis
          condimentum augue id magna semper rutrum. Nam quis nulla. Etiam commodo dui eget wisi. Fusce suscipit libero
          eget elit. Etiam bibendum elit eget erat. Vestibulum erat nulla, ullamcorper nec,rutrum non, nonummy ac, erat.
          Etiam neque. Vivamus luctus egestas leo. Integer in sapien. Nullam sapien sem, ornare ac, nonummy non,
          lobortis a enim. Donec quis nibh at felis congue commodo. Pellentesque arcu. In enim a arcu imperdiet
          malesuada. Duis bibendum, lectus ut viverra rhoncus, dolor nunc faucibus libero, eget facilisis enim ipsum id
          lacus. Cras pede libero, dapibus nec, pretium sit amet, tempor quis. Quis autem vel eum iure reprehenderit qui
          in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas
          nulla pariatur? Maecenas fermentum, sem in pharetra pellentesque, velit turpis volutpat ante, in pharetra
          metus odio a lectus. Duis sapien nunc, commodo et, interdum suscipit, sollicitudin et, dolor. Fusce aliquam
          vestibulum ipsum. Duis ante orci, molestie vitae vehicula venenatis, tincidunt ac pede. Aenean id metus id
          velit ullamcorper pulvinar.
        </UU5.Bricks.Newspaper>
      </UU5.Bricks.Container>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
