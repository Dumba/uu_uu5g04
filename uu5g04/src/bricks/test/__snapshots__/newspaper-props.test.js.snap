// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UU5.Bricks.Newspaper checkGrammar=false 1`] = `
<Section
  checkGrammar={false}
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper checkGrammar=true 1`] = `
<Section
  checkGrammar={true}
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper checkHighlight=false 1`] = `
<Section
  checkHighlight={false}
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper checkHighlight=true 1`] = `
<Section
  checkHighlight={true}
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper checkSpaces=false 1`] = `
<Section
  checkSpaces={false}
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper checkSpaces=true 1`] = `
<Section
  checkSpaces={true}
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper className="my-classname" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text my-classname uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="brown" 1`] = `
<Section
  className="color-schema-brown uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="brown-rich" 1`] = `
<Section
  className="color-schema-brown-rich uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="cyan" 1`] = `
<Section
  className="color-schema-cyan uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="cyan-rich" 1`] = `
<Section
  className="color-schema-cyan-rich uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="default" 1`] = `
<Section
  className="color-schema-default uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="green" 1`] = `
<Section
  className="color-schema-green uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="green-rich" 1`] = `
<Section
  className="color-schema-green-rich uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="grey" 1`] = `
<Section
  className="color-schema-grey uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="grey-rich" 1`] = `
<Section
  className="color-schema-grey-rich uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="orange" 1`] = `
<Section
  className="color-schema-orange uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="orange-rich" 1`] = `
<Section
  className="color-schema-orange-rich uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="pink" 1`] = `
<Section
  className="color-schema-pink uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="pink-rich" 1`] = `
<Section
  className="color-schema-pink-rich uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="purple" 1`] = `
<Section
  className="color-schema-purple uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="purple-rich" 1`] = `
<Section
  className="color-schema-purple-rich uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="red" 1`] = `
<Section
  className="color-schema-red uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="red-rich" 1`] = `
<Section
  className="color-schema-red-rich uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="yellow" 1`] = `
<Section
  className="color-schema-yellow uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper colorSchema="yellow-rich" 1`] = `
<Section
  className="color-schema-yellow-rich uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper columnsCount=1 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-1"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper columnsCount=2 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper columnsCount=3 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-3"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper columnsCount=4 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-4"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper columnsCount=5 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-5"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper columnsCount=6 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-6"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper content="This is Content from ContentMixin" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  content="This is Content from ContentMixin"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper content=12345 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  content={12345}
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper controlled=false 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper controlled=true 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper default props 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper disabled=false 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper disabled=true 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  disabled={true}
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper docKit examples UU5.Bricks.Newspaper should render without crash 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="iiROOT"
  style={Object {}}
  title={null}
>
  <Newspaper
    columnsCount={3}
    id="uuID"
    key="uuID"
  >
    Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas libero. Aliquam erat volutpat. Duis condimentum augue id magna semper rutrum. Nam quis nulla. Etiam commodo dui eget wisi. Fusce suscipit libero eget elit. Etiam bibendum elit eget erat. Vestibulum erat nulla, ullamcorper nec,rutrum non, nonummy ac, erat. Etiam neque. Vivamus luctus egestas leo. Integer in sapien. Nullam sapien sem, ornare ac, nonummy non, lobortis a enim. Donec quis nibh at felis congue commodo. Pellentesque arcu. In enim a arcu imperdiet malesuada. Duis bibendum, lectus ut viverra rhoncus, dolor nunc faucibus libero, eget facilisis enim ipsum id lacus. Cras pede libero, dapibus nec, pretium sit amet, tempor quis. Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur? Maecenas fermentum, sem in pharetra pellentesque, velit turpis volutpat ante, in pharetra metus odio a lectus. Duis sapien nunc, commodo et, interdum suscipit, sollicitudin et, dolor. Fusce aliquam vestibulum ipsum. Duis ante orci, molestie vitae vehicula venenatis, tincidunt ac pede. Aenean id metus id velit ullamcorper pulvinar.
  </Newspaper>
</div>
`;

exports[`UU5.Bricks.Newspaper dynamic=false 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper dynamic=true 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  dynamic={true}
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper footer={"type":"div","key":null,"ref":null,"props":{"children":"Footer of section"},"_owner":null,"_store":{}} 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  footer={
    <div>
      Footer of section
    </div>
  }
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper header={"type":"div","key":null,"ref":null,"props":{"children":"Header of section"},"_owner":null,"_store":{}} 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  header={
    <div>
      Header of section
    </div>
  }
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper hidden=false 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper hidden=true 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  hidden={true}
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper id="myId" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="myId-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper ignoreHTML=false 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper ignoreHTML=true 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level="0" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level="0"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level="1" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level="1"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level="2" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level="2"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level="3" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level="3"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level="4" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level="4"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level="5" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level="5"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level="6" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level="6"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level=0 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level={0}
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level=1 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level={1}
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level=2 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level={2}
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level=3 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level={3}
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level=4 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level={4}
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level=5 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level={5}
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper level=6 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  level={6}
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper mainAttrs=undefined 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper name="myName" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  name="myName-inner"
/>
`;

exports[`UU5.Bricks.Newspaper nestingLevel="bigBox" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  nestingLevel="bigBox"
/>
`;

exports[`UU5.Bricks.Newspaper nestingLevel="bigBoxCollection" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  nestingLevel="bigBoxCollection"
/>
`;

exports[`UU5.Bricks.Newspaper nestingLevel="box" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  nestingLevel="box"
/>
`;

exports[`UU5.Bricks.Newspaper nestingLevel="boxCollection" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  nestingLevel="boxCollection"
/>
`;

exports[`UU5.Bricks.Newspaper nestingLevel="container" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  nestingLevel="container"
/>
`;

exports[`UU5.Bricks.Newspaper nestingLevel="inline" 1`] = `""`;

exports[`UU5.Bricks.Newspaper nestingLevel="page" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  nestingLevel="page"
/>
`;

exports[`UU5.Bricks.Newspaper nestingLevel="smallBox" 1`] = `""`;

exports[`UU5.Bricks.Newspaper nestingLevel="smallBoxCollection" 1`] = `""`;

exports[`UU5.Bricks.Newspaper nestingLevel="spa" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  nestingLevel="spa"
/>
`;

exports[`UU5.Bricks.Newspaper noIndex=false 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper noIndex=true 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-noindex uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper pureRender=false 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper pureRender=true 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  pureRender={true}
/>
`;

exports[`UU5.Bricks.Newspaper selected=false 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper selected=true 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  selected={true}
/>
`;

exports[`UU5.Bricks.Newspaper style="padding:8px; borderRadius: 2px" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
/>
`;

exports[`UU5.Bricks.Newspaper style={"backgroundColor":"red","color":"blue"} 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
/>
`;

exports[`UU5.Bricks.Newspaper textCorrector=false 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  textCorrector={false}
/>
`;

exports[`UU5.Bricks.Newspaper textCorrector=true 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
  textCorrector={true}
/>
`;

exports[`UU5.Bricks.Newspaper tooltip="My Tooltip" 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={
    Object {
      "title": "My Tooltip",
    }
  }
  tooltip="My Tooltip"
/>
`;

exports[`UU5.Bricks.Newspaper underline=false 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;

exports[`UU5.Bricks.Newspaper underline=true 1`] = `
<Section
  className="uu5-bricks-newspaper uu5-common-text uu5-common-newspaper-layout-2"
  id="uuID-inner"
  mainAttrs={Object {}}
/>
`;
